lib/open.js

```diff
@@ -34,7 +34,7 @@ function makeArguments(filename, settings) {
     //   => "filename:1:2"
     //
     .replace(/\{filename\}(\S*)/, function(m, rest) {
-      return quote(info.filename + rest, settings.escapeQuotes);
+      return quote(info.filename, settings.escapeQuotes) + rest;
```

Comment:
Ideally it should be "filename:{line}:{column}". Would it be acceptable if I refactor to this?